Streetcode/Streetcode.BLL/MediatR/Authentication/Login/LoginHandler.cs

```diff
@@ -62,7 +62,7 @@ public async Task<Result<LoginResponseDTO>> Handle(LoginQuery request, Cancellat
                     var stringToken = new JwtSecurityTokenHandler().WriteToken(token);
 
                     var userDTO = _mapper.Map<UserDTO>(user);
-                    string? userRole = (await _userManager.GetRolesAsync(user !)).FirstOrDefault();
+                    string userRole = (await _userManager.GetRolesAsync(user !)).FirstOrDefault();

```

Comment:
Don't quite get why this was needed because `FirstOrDefault` can potentially return `null` but if you sure what you do then I think you don't need this null check on the next line `userDTO.Role = userRole ?? "User"` and maybe need to change `First` instead of `FirstOrDefault`